user/model/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Don’t forget to import __init__
```

Comment:
This is only needed where we could not use Poetry such as CTN's Apple M1 MacBook. TODO: investigate, fix Poetry issues for that case and remove this __init__.py file.

Reply:
Yep.

Reply:
Another workaround is to not develop on M1!